### PR TITLE
Version bump and GitHub Actions

### PR DESCRIPTION
**Pull request by lmweber**
PR to add a version bump and update GitHub Actions (Linux only for now)

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: SpatialExperiment
-Version: 1.7.3
+Version: 1.9.1
```
Comment (drighelli) [inline DESCRIPTION:2]: why so many versions?

**Comment by lmweber** [comment]
Hey, I should have added a comment above.

This is to (i) merge the version bump commits from the Bioc team (after Bioc-release) into our master branch on GitHub, and (ii) an additional version bump for Aaron's PR last week. Also (iii) update GitHub Actions to use the latest Bioc-devel in our master branch.

I double-checked just now and the version number looks right. Currently on the Bioc landing pages we have version 1.8.0 for release (Bioc 3.16) and 1.9.0 for devel (Bioc 3.17). So now devel / master branch will be 1.9.1 with the version bump above.

Thanks!

**Comment by drighelli** [comment]
Nice, Thanks!!!

**Review by drighelli — APPROVED** [review]

**Comment by lmweber** [comment]
Thanks! I was thinking the GHA fail may be temporary due to the Bioc release cycle. Maybe the Bioc docker images aren't available yet for the new release and devel. In this case maybe best to just wait a week or so and see if it resolves.

I'll monitor it though in case this is not true and we need to fix something.

**Pull request merged**